src/js/components/Anchor/Anchor.js

```diff
@@ -73,8 +60,11 @@ class Anchor extends Component {
         href={!disabled ? href : undefined}
         onClick={!disabled ? onClick : undefined}
       >
-        {first}
-        {second}
+        {(first || second) ? (
+          <Box direction='row' align='center' gap='small'>
```

Comment:
I don't think this works correctly when including an Anchor in a Paragraph. For instance, I would expect the follow to be equal:
`<Paragraph size='large'>Lorem ipsum <Anchor href='#'>Add</Anchor> dolor est.</Paragraph>`
<img width="344" alt="screen shot 2018-06-11 at 1 51 37 pm" src="https://user-images.githubusercontent.com/11637956/41256498-a0c69b04-6d7e-11e8-9a89-387de3cab455.png">

`<Paragraph size='large'>Lorem ipsum <Anchor href='#' label='Add' /> dolor est.</Paragraph>`
<img width="172" alt="screen shot 2018-06-11 at 1 29 49 pm" src="https://user-images.githubusercontent.com/11637956/41256505-a5c0f41a-6d7e-11e8-8d30-c5fb2557fa65.png">